src/mapbox_logo.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var MapboxLogo = L.Control.extend({
+
+    options: {
+        position: 'bottomleft',
+        sanitizer: require('sanitize-caja')
+    },
+
+    initialize: function(_) {
+        this._mapboxLogo = '';
+        this._ = _;
```

Comment:
what is _? can this be renamed options? Also, note that the L.Control class has a `.setOptions` method that this should probably use instead